appveyor.yml

```diff
@@ -1,8 +1,7 @@
 environment:
   matrix:
-  - julia_version: 0.6
-  - julia_version: 0.7
   - julia_version: 1.0
+  - julia_version: 1.1
```

Comment:
I'll note 1.2-rc1 just got released yesterday so we can also add: `julia_version: 1.2`